### ISSUE TITLE
Error when trying to schedule a report

### ISSUE THREAD

**Issue by jackbenny** [Icinga/icingaweb2-module-reporting#41]
## Describe the bug
After a report is created, when selecting **Schedule** to schedule the report, it fails with the following error message:

```
Uncaught Error: Class 'Icinga\Module\Reporting\DivDecorator' not found in /usr/share/icingaweb2/modules/reporting/library/Reporting/Web/Forms/ScheduleForm.php:56
Stack trace:
#0 /usr/share/icingaweb2/modules/ipl/vendor/ipl/html/src/HtmlDocument.php(258): Icinga\Module\Reporting\Web\Forms\ScheduleForm->assemble()
#1 /usr/share/icingaweb2/modules/ipl/vendor/ipl/html/src/Form.php(61): ipl\Html\HtmlDocument->ensureAssembled()
#2 /usr/share/icingaweb2/modules/reporting/application/controllers/ReportController.php(96): ipl\Html\Form->handleRequest(Object(GuzzleHttp\Psr7\ServerRequest))
#3 /usr/share/icingaweb2/library/vendor/Zend/Controller/Action.php(507): Icinga\Module\Reporting\Controllers\ReportController->scheduleAction()
#4 /usr/share/php/Icinga/Web/Controller/Dispatcher.php(76): Zend_Controller_Action->dispatch('scheduleAction')
#5 /usr/share/icingaweb2/library/vendor/Zend/Controller/Front.php(937): Icinga\Web\Controller\Dispatcher->dispatch(Object(Icinga\Web\Request), Object(Icinga\Web\Response))
#6 /usr/sh
```

## To Reproduce

1. Create a report (Reports -> New Report -> Fill out _name_, _timeframe_ and select _Host SLA_ for type and click Create Report)

2. Select the report from the list

3. Click on **Schedule** on the top of the page

## Expected behavior
A page with fields to fill out for the schedule.

## Screenshots
![bild](https://user-images.githubusercontent.com/1704592/67947433-f5b5ef80-fbe3-11e9-867b-8e611fc9317e.png)

![bild](https://user-images.githubusercontent.com/1704592/67946599-f6e61d00-fbe1-11e9-9206-dc693536546b.png)

## Your Environment

* Module version: 0.9.1
* Dependent module versions: 
    * idoreports 0.9.1
    * ipl 0.3.0
    * pdfexport 0.9.1
    * reactbundle 0.7.0
* Icinga Web 2 version and modules (System - About): 2.7.3

```
businessprocess 2.2.0
director 1.7.0
graphite 1.1.0
idoreports 0.9.1
incubator 0.5.0
ipl v0.3.0
monitoring 2.7.3
pdfexport 0.9.1
reactbundle 0.7.0
reporting 0.9.1
vspheredb 1.0.4
```

* Web browser and version: Firefox 70.0
* Icinga 2 version used (`icinga2 --version`): r2.11.1-1
* PHP version used (`php --version`): 7.2.24
* Server operating system and version: Ubuntu 18.04.3

**Comment by dnsmichi** [comment]
That's fixed with #28 and release 0.9.2. Please update.

**Comment by jackbenny** [comment]
@dnsmichi Thank you, that fixed it! Actually there is no version 0.9.2 tagged, but I did a checkout on the commit with the comment _Raise version to 0.9.2_. I think you might have forgotten to tag it?

**Comment by dnsmichi** [comment]
Looks like that. @lippserd please take care.
